src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Selection.js

```diff
@@ -140,6 +143,22 @@ class Selection extends React.Component<Props> {
 
                 return change;
             });
+        } else if (this.type === 'auto_complete') {
+            const {value} = this.props;
+
+            this.selectionStore = new MultiSelectionStore(
+                resourceKey,
+                value || [],
+                this.locale,
+                this.autoCompleteFilterParameter
+            );
+
+            this.changeSelectionDisposer = reaction(
+                () => this.selectionStore
+                    ? this.selectionStore.items.map((item) => item[this.autoCompleteIdProperty])
+                    : [],
+                this.handleSelectionChange
```

Comment:
also this handles only one direction. you are not updating items of the `SelectionStore` of the `value` of this component is changed at the moment

Reply:
That's possible... Gonna check that as well.